src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arr = [];
+
+  for (const i of actions) {
+    let item = {};
+
+    if (arr[0] === undefined) {
+      item = { ...state };
+    } else {
+      item = { ...arr[arr.length - 1] };
+    };
```

Comment:
```suggestion
    let item = { ...state };
```
Змінну з копією стейту потрібно створити поза межами циклу. 
І тобі не потрібна буде ось ця перевірка